File: dehaze-go/model/sys_wpx_file.go
package model

// SysWpxFile WPX文件表
type SysWpxFile struct {
	BaseModel
	OriginFileID int64  `gorm:"column:origin_file_id;type:bigint;not null;comment:原始文件ID" json:"originFileId"`
	OriginMD5    string `gorm:"column:origin_md5;type:char(32);not null;comment:原始文件MD5" json:"originMd5"`
	OriginPath   string `gorm:"column:origin_path;type:varchar(255);not null;comment:原始文件路径" json:"originPath"`
	NewFileID    int64  `gorm:"column:new_file_id;type:bigint;not null;comment:新文件ID" json:"newFileId"`
	NewPath      string `gorm:"column:new_path;type:varchar(255);not null;comment:新文件路径" json:"newPath"`
	NewMD5       string `gorm:"column:new_md5;type:char(32);not null;comment:新文件MD5" json:"newMd5"`
}

File: dehaze-python/app/models/entity/sys_wpx_file.py
"""
WPX 文件映射实体（对齐 Java SysWpxFile）
"""

from typing import Optional

from sqlalchemy import BigInteger, CHAR, Index, String
from sqlalchemy.orm import Mapped, mapped_column

from app.models.base import BaseModel


class SysWpxFile(BaseModel):
    __tablename__ = 'sys_wpx_file'
    __table_args__ = (
        Index('idx_origin_md5', 'origin_md5'),
        {'comment': 'WPX 文件映射表'},
    )

    id: Mapped[int] = mapped_column(
        BigInteger, primary_key=True, autoincrement=True, comment='id')
    origin_file_id: Mapped[Optional[int]] = mapped_column(
        BigInteger, nullable=True, comment='旧文件id')
    origin_md5: Mapped[str] = mapped_column(
        CHAR(32), nullable=False, unique=True, comment='旧文件的MD5值')
    origin_path: Mapped[str] = mapped_column(
        String(255), nullable=False, comment='旧文件路径')
    new_file_id: Mapped[Optional[int]] = mapped_column(
        BigInteger, nullable=True, comment='新文件id')
    new_path: Mapped[str] = mapped_column(
        String(255), nullable=False, comment='新文件路径')
    new_md5: Mapped[str] = mapped_column(
        CHAR(32), nullable=False, unique=True, comment='新文件的MD5值')
